features/course_choices.feature

```diff
@@ -0,0 +1,26 @@
+@candidate
+Feature: same & different course choices
+
+  Candidates shouldn't be able to apply to the same course multiple times.
+
+  # TODO: are choices to same provider/same course/different accredited body
+  # the same or different?
+
+  Background:
+    Given the following providers:
+      | provider code | provider training locations |
+      | U80           | A, B                        |
+      | S13           | A                           |
+    And the following courses:
+      | provider code | course code | course training locations |
+      | U80           | X100        | A, B                      |
+
+  Scenario Outline: same & different courses
+    Then <course choice A> and <course choice B> are treated as the same choice: <same?>
+
+  Examples:
+    | course choice A        | course choice B        | same? | notes                                        |
+    | U80/X100 (location: A) | S13/372B (location: A) | N     | Different providers                          |
```

Comment:
Courses 372B and A123 are not set up in the "courses" background step. This might not be technically necessary but it's still a bit of grit to work through when reading the example table.

Reply:
Thanks - fixed on #186